### PR TITLE
added: grids-simplified, variables, styles-imports updated

### PR DESCRIPTION
**Pull request by rishbitsnbytes**
Added Simplified Grids Component.

Grid 50-50
Grid 30-70
Grid 70-30

Grid 4 col layout for product listing.

Previews : 

grids 50-50,30-70,70-30

grid-4-col

### CODE REVIEW

**Review by Sharmaryan — APPROVED** [review]
UI is cool, and code is also good

File: Components/Simplified-Grid/grids.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
+@import url(/Styles/variables-and-fonts.css);
+@import url(/Styles/all-imports.css);
+
```
Comment (Sharmaryan) [inline Components/Simplified-Grid/grids.css:3]: space here

File: Components/Simplified-Grid/grids.html [hunk 2]
```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Rishabh Rathore">
```
Comment (Sharmaryan) [inline Components/Simplified-Grid/grids.html:8]: although not necessary, but you can also add meta tag for description

**Review by Sharmaryan — APPROVED** [review]

File: Components/Simplified-Grid/grids.html [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+            <img class="img-responsive" src="/Assets/boysShoesLS.jpg">
+        </div>
+        <div class="text-container">
+            <h1 class="text-container-title">King of Shoes</h1>
+            <h3 class="text-container-desc">Best Value @ Rs.999</h3>
+            <a class="btn btn-primary">Explore</a>
+        </div>
+    </div>
+
+
```
Comment (bharati-21) [inline Components/Simplified-Grid/grids.html:53]: You can maintain space consistency.

File: Components/Images/images.css [hunk 4]
```diff
@@ -7,7 +7,7 @@
 .img-responsive {
```
Comment (bharati-21) [inline Components/Images/images.css:7]: It's a suggestion to not change styles of other components while working on the branch of another one.

**Review by bharati-21 — APPROVED** [review]
Looks good to be merged!

File: Components/Images/images.css (hunk 4 above)
Comment (rishbitsnbytes) [reply Components/Images/images.css:7]: Thanks Bharati for mentioning this. I will keep this in mind.

**Pull request merged**